src/main/java/com/juansecu/opentoonix/mail/VerificationTokenEntity.java

```diff
@@ -0,0 +1,36 @@
+package com.juansecu.opentoonix.mail;
+
+/* --- Java modules --- */
+import java.util.Date;
+
+/* --- Javax modules --- */
+import javax.persistence.*;
+
+/* --- Third-party modules --- */
+import lombok.Data;
+import org.hibernate.annotations.GenericGenerator;
+
+
+/* --- Application modules --- */
+import com.juansecu.opentoonix.user.models.entities.UserEntity;
+import com.juansecu.opentoonix.shared.generators.NumericIdGenerator;
```

Comment:
Be sure to import modules alphabetically. If you're using an IDE like IntelliJ, it makes that for you, but make sure to separate the application modules from the Java, Javax or third-party ones.